docs/monitoring/metrics.zh.md

```diff
@@ -667,10 +668,11 @@ metrics.reporter.influxdb.port: 8086
 metrics.reporter.influxdb.db: flink
 metrics.reporter.influxdb.username: flink-metrics
 metrics.reporter.influxdb.password: qwerty
+metrics.reporter.influxdb.retentionPolicy: one_hr
```

Comment:
same as above: use "one_hour" as in the tests?

Reply:
Updated based on recommendation.